Makefile

```diff
@@ -166,7 +168,7 @@ sam-publish-all:
 build-App:
 	$(call check_var,APP)
 	$(call check_var,ARTIFACTS_DIR)
-	GOARCH=arm64 GOOS=linux go build -tags lambda.norpc -o ./bootstrap cmd/$(APP)/main.go
+	GOARCH=arm64 GOOS=linux go build -tags lambda.norpc -ldflags "-X $(shell go list -m)/version.Version=${VERSION}" -o ./bootstrap cmd/$(APP)/main.go
```

Comment:
what does `shell go list -m` output?

Reply:
The package root, `github.com/observeinc/aws-sam-apps`